dv/uvm/icache/dv/ibex_icache_sim.core

```diff
@@ -0,0 +1,25 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:ibex_icache_sim:0.1"
```

Comment:
Please put this into the `lowrisc:ibex` namespace.

Reply:
Will do.